src/code42cli/main.py

```diff
@@ -24,8 +25,11 @@
 from code42cli.cmds.shell import shell
 from code42cli.cmds.trustedactivities import trusted_activities
 from code42cli.cmds.users import users
+from code42cli.cmds.watchlists import watchlists
 from code42cli.options import sdk_options
 
+warnings.simplefilter("ignore", DeprecationWarning)
```

Comment:
Do we want deprecation warnings off by default?

Reply:
Yep, otherwise the user would get deprecation warnings to stderr for the py42 method names, which could be confusing.